pkg/connectors/gitlabmr/connector.go

```diff
@@ -102,6 +117,21 @@ func (c *Connector) collectMRs(ctx context.Context) ([]mergeRequest, error) {
 	return mrs, nil
 }
 
+func (c *Connector) collectGroupMRs(ctx context.Context) ([]mergeRequest, error) {
```

Comment:
We talked about merging the two collectXYMRs, but individually decided that there is no reason to optimize this code and make it less readable.